purchase_order_sending_warning/models/purchase_order.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    sending_warning = fields.Selection(selection=[("email_not_sent", "Email not sent")])
+
+    error_in_sending = fields.Boolean(
+        store=True,
+        compute="_compute_error_in_sending",
+        string="Error in sending",
+    )
```

Comment:
I wonder if we need 2 fields here. Maybe the Selection field is enough.


For the name of the field, I would prefer `transmission_error` or `sending_error` since it is more important than a warning.

Reply:
@florentx 2 fields are there because selection is going to be extended (for example by edi) and user would like to know POs which failed (whatever they failed - edi or email), so computed field will be set whenever mail/edi sending fails. I'll rename.